<?xml version="1.0"?>
<launch>
  
  <arg name="serial" default="160678" />
  
  <!-- Ensenso node -->
  <node name="ensenso_node" pkg="ensenso" type="ensenso_node" ns="ensenso" output="screen" required="true"> 
    <param name="serial"          type="string" value="$(arg serial)" />
    <param name="camera_frame_id" type="string" value="ensenso_optical_frame" />
    <param name="config_json_file" value="$(find ensenso)/config/ensenso_config.json"/>
    <param name="point_cloud"     type="bool"   value="true" />
    <param name="front_light"     type="bool"   value="true" />
    <param name="projector"       type="bool"   value="true" />
  </node>
  <!-- RViz -->
  <node name="rviz_node" pkg="rviz" type="rviz" args="-d $(find ensenso)/config/viewer.rviz" />
  
</launch>
